manual/cling/index.md

```diff
@@ -127,18 +127,16 @@ filename     line:size busy function type and name
 (compiled)     (NA):(NA) 0 public: static int ImplFileLine();
 (compiled)     (NA):(NA) 0 public: static const char *ImplFileName();
 (compiled)     (NA):(NA) 0 public: static int DeclFileLine();
+root [] TLine l;
 root [] .> test.log
-root [] l.Dump();
+ l.Dump();
 root [] .>
-root [] ?
 {% endhighlight %}
 
 Here we see:
 
 -   Use `.class` as quick help and reference
--   Unix like I/O redirection using `.> out.txt` and unredirection with `.>`
--   Use `?` to get help on all ‘‘raw'' interpreter commands
--   Use @ to abort a multi-line command
```

Comment:
This one is still valid, by changing to `.@`

Reply:
This PR is correct now. Do you agree ? I checked it . the TLine `l`was not not created either....
Do you agree I merge it ?

Reply:
I would restore line 141, but changing @ with .@
.@ works well for interrupting a multi-line command.

Reply:
I remove it because it is better explained in the next example

Reply:
oh I see.

And what about removing the number within the root[ ] brackets?